### PR TITLE
Test for add and remove function

### PR DESCRIPTION
**Pull request by Uchenna-ibute**
Create a test file to test the add item and delete item functions.

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Uchenna-ibute and @gsmalik030, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Invalid

You have submitted a project with modified linters configuration. Kindly use the correct version of the `.eslintrc.json` file in the [linters configuration](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -87,7 +88,7 @@ git - git should be installed in local machine. git is necessary for cloning thi
 
 Clone this repository to your desired folder:
 
-git clone https://github.com/Uchenna-ibute/Awesome-book-es6.git
+git clone https://https://github.com/Uchenna-ibute/To-do-list.git
 
 ### Install
```
Comment (t-yanick) [inline README.md:86]: - [ ] To make your ReadMe file more professional write to commands to `install`, `run` your app and also to run the `tests`. You just have the headings with no commands. Kindly add them

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Uchenna-ibute and @gsmalik030 , 

### Good job so far!

### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive PR

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @Uchenna-ibute 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**